tegra/tegra.c

```diff
@@ -240,11 +239,9 @@ int drm_tegra_bo_unref(struct drm_tegra_bo *bo)
 	if (!atomic_dec_and_test(&bo->ref))
 		return 0;
 
-	drm = bo->drm;
-
 	pthread_mutex_lock(&table_lock);
 
-	if (!bo->reuse || (drm_tegra_bo_cache_free(&drm->bo_cache, bo) != 0))
+	if (!bo->reuse || drm_tegra_bo_cache_free(&bo->drm->bo_cache, bo))
```

Comment:
Or maybe it's just about making the code more consistent? Either way, doesn't seem to matter much.